src/coefficients/CMakeLists.txt

```diff
@@ -6,11 +6,13 @@
 
 set(coefficients_sources
     loading_functions.cpp
+    StdFunctionCoefficient.cpp
     )
 
 set(coefficients_headers
     loading_functions.hpp
     traction_coefficient.hpp
+    StdFunctionCoefficient.hpp
```

Comment:
Please follow the file name conventions of the repo. We can discuss what we should be using if anyone cares but please be consistent.

Reply:
Alright. I will make the file names pot_hole_case. 

FYI. It seems at the moment the repo uses pot_hole_case for file names, camel_Pot_hole for member names, and UpperCamelCase for Classes and Method names. I would not mind making it further consistent.

Reply:
Yeah, I don't really care what we settle on as long as its consistent. I think I tried to come up with standards, but I probably missed some stuff. I don't really have opinions on one convention over another.